opamp/config_test.go

```diff
@@ -188,15 +188,17 @@ func TestToTLS(t *testing.T) {
 				}
 
 				expectedConfig := tls.Config{
-					MinVersion: tls.VersionTLS12,
+					// MinVersion: tls.VersionTLS12,
```

Comment:
Was there an issue w/ this test locally, commented out temporarily?

Reply:
My IDE simply flagged the sections I've commented out as unused - they're set, but then never referenced again (in this particular test, the only `require.Equal` call just references `expectedConfig.Certificates`, and the rest of `expectedConfig` is ignored). My take was that commenting out the unused aspects was a middle ground between simply removing all the unused sections (which might be valuable in the future, if this test is ever improved to account for them) and letting the scope of this ticket creep to actually improve the test now. Happy to do either of them if your take is different from mine.

Reply:
We synced n this offline :+1: